plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -64,6 +64,12 @@ const { render, renderCompat } = (() => {
     }
   }
 
+  if (typeof additionalTemplateGlobals !== 'undefined') {
+    for (const [globalName, globalFn] of Object.entries(additionalTemplateGlobals)) {
+      env.addGlobal(globalName, (...args) => JSON.parse(globalFn(...args)));
```

Comment:
These globals can be either functions or values right? I'm feeling we should add a bit of `typeof x ===  'function'` logic to make it possible to pass both JSON values as well as functions that return JSON values.